kata-webhook/deploy/webhook.yaml

```diff
@@ -33,6 +33,10 @@ spec:
             - name: webhook-certs
               mountPath: /etc/webhook/certs
               readOnly: true
+          resources:
```

Comment:
seems fine -- any particular reason for 1 CPU instead of 100m, or 100 CPU? 

Same for memory

Reply:
@egernst for no particular reason. But maybe it should be better set 100m?

Reply:
Just updated this PR; setting "cpu==100m" and "memory==205Mi", which should be enough.